docs/source/upcoming_release_notes/1327-LasBasler_User_Presets.rst

```diff
@@ -0,0 +1,31 @@
+1327 LasBasler User Presets
+#################
+
+API Breaks
+----------
+- N/A
+
+Library Features
+----------------
+- N/A
+
+Device Features
+---------------
+- LasBasler: Adds long_names to signals
+- LasBasler: Adds UserPresetSelector and UserPresetDefaultSe signals
```

Comment:
```suggestion
- LasBasler: Adds `save_setting` and `load_setting` signals for saving and loading settings
```

I think it's maybe more helpful to mention the python attributes people will be using.  Also "UserPresetSelector" does not exist in this PR